lib/cisco_node_utils/interface.rb

```diff
@@ -96,6 +97,22 @@ def default_description
       config_get_default('interface', 'description')
     end
 
+    def enable_pim_sparse_mode
+      state = config_get('interface', 'pim_sparse_mode', @name)
+      state ? true : false
+    end
```

Comment:
1. This should be `pim_sparse_mode` or possibly `ip_pim_sparse_mode`
2. There needs to be a feature pim check
3. You don't need the state var.

```
def pim_sparse_mode
  return false unless Pim.feature_enabled
  config_get('interface', 'pim_sparse_mode', @name)
end
```